plugins/automation/admin/assets/javascripts/admin/routes/admin-plugins-discourse-automation-index.js

```diff
@@ -1,9 +1,22 @@
 import { action } from "@ember/object";
+import { service } from "@ember/service";
 import DiscourseRoute from "discourse/routes/discourse";
 
 export default class AutomationIndex extends DiscourseRoute {
+  @service router;
+
   controllerName = "admin-plugins-discourse-automation-index";
 
+  afterModel(model) {
```

Comment:
Oh tiny nitpick, but it would be good to have the `afterModel` hook after the `model` hook. OCD, sorry!